featurebyte/models/feature_list_store_info.py

```diff
@@ -178,26 +178,38 @@ def feature_specs_definition(self) -> str:
         return codes
 
     @staticmethod
-    def _create_feature_functions(features: List[FeatureModel]) -> List[DataBricksFeatureFunction]:
+    def _get_fully_qualified_schema_name(feature_store: FeatureStoreModel) -> str:
+        feature_store_details = feature_store.details
+        assert isinstance(feature_store_details, DatabricksDetails)
+        catalog_name = feature_store_details.catalog_name
+        schema_name = feature_store_details.schema_name
+        return f"{catalog_name}.{schema_name}"
+
+    @staticmethod
+    def _create_feature_functions(
+        features: List[FeatureModel], schema_name: str
+    ) -> List[DataBricksFeatureFunction]:
         feature_functions = []
         for feature in features:
             offline_store_info = feature.offline_store_info
             if offline_store_info.udf_info:
                 input_bindings = {
-                    input_arg_name: input_info.column_name
-                    for input_arg_name, input_info in offline_store_info.udf_info.input_var_name_to_info.items()
+                    sql_input_info.sql_input_var_name: sql_input_info.column_name
+                    for sql_input_info in offline_store_info.udf_info.sql_inputs_info
                 }
                 feature_functions.append(
                     DataBricksFeatureFunction(
-                        udf_name=offline_store_info.udf_info.sql_function_name,
+                        udf_name=f"{schema_name}.{offline_store_info.udf_info.sql_function_name}",
```

Comment:
Curious when do we need fully qualified name, or is schema qualified is sufficient sometimes?

Reply:
I did not try schema qualified. But function name alone is not sufficient.

Reply:
It does work for schema qualified. However, if we add backticks, it fails:
```
Exception: {'error_code': 'RESOURCE_DOES_NOT_EXIST', 'message': "Feature table 'ml.`FEATUREBYTE_SENBONG`.fb_entity_grocerycustomerguid_659f7dbbd1f681429083addb' does not exist."}
```

Reply:
I thought the current version

```
"{schema_name}.{offline_store_info.udf_info.sql_function_name}"
```

is schema qualified, as opposed to fully qualified

```
"{catalog_name}.{schema_name}.{offline_store_info.udf_info.sql_function_name}"
```

hence the question.

Reply:
Current version is fully qualify as the `schema_name` here contains `catalog_name`.
```
    def _get_fully_qualified_schema_name(feature_store: FeatureStoreModel) -> str:
        feature_store_details = feature_store.details
        assert isinstance(feature_store_details, DatabricksDetails)
        catalog_name = feature_store_details.catalog_name
        schema_name = feature_store_details.schema_name
        return f"{catalog_name}.{schema_name}"
```

Reply:
Got it, I thought it was only the schema name 👍